airflow/cli/cli_parser.py

```diff
@@ -120,15 +120,21 @@ def add_to_parser(self, parser: argparse.ArgumentParser):
         parser.add_argument(*self.flags, **self.kwargs)
 
 
-def positive_int(value):
+def positive_int(*, allow_zero):
     """Define a positive int type for an argument."""
-    try:
-        value = int(value)
-        if value > 0:
-            return value
-    except ValueError:
-        pass
-    raise argparse.ArgumentTypeError(f"invalid positive int value: '{value}'")
+
+    def _check(value):
+        try:
+            value = int(value)
+            if allow_zero and value == 0:
+                return True
+            if value > 0:
+                return value
```

Comment:
The return value is either bool or an int - is this inconsistency expected?

Reply:
Fixed. Thanks